### DISCUSSION TITLE
Is Email Sender considered by Google Play to collect data?

### DISCUSSION THREAD

**Discussion by ankandrew** [ACRA/acra#1122] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/61120139/200417010-9cb62015-c516-4e73-902a-4c38e7ab4094.png)

I'm using ACRA latest version (`5.9.6`) and trying to fill in the required forms to publish an app. I'm filling Data Safety section in Google Play Console, however I'm unsure what to answer.

More specifically my doubt comes after reading:

> On the next screen, you'll be asked whether your app collects or shares any of the required user data types. [View required data types](https://support.google.com/googleplay/android-developer/answer/10787469?hl=en#types)

If you see [required data types](https://support.google.com/googleplay/android-developer/answer/10787469?hl=en#) crash logs are mentioned:

![image](https://user-images.githubusercontent.com/61120139/200417579-9777df32-9d40-4ed4-8e55-2b2738d760aa.png)

So when asked:

![image](https://user-images.githubusercontent.com/61120139/200417897-61627dfc-291e-44cb-9d68-dc43008ec396.png)

Should I answer yes? I'm using the email sender, so nothing gets sent without the user consent.

**Comment by F43nd1r** [comment, marked as answer]
I would say the argument is, the user shares the crash report, not the app. Imo that means answering no would be okay when using the email sender. The answer is yes for sure with a http sender.

_Disclaimer: I am not a lawyer and this is not legal advice_

**Reply by ankandrew** [reply]
Thanks for the quick reply, I think 'no' should be acceptable. Let's see how it goes

**Reply by ankandrew** [reply]
No problems here with email sender and answering 'no'
